docs/DEPLOYMENT.md

```
@@ -0,0 +1,380 @@
+### STEP-2) Set up development environment variables
+
+1. Create a copy of the example environment file and open it for editing:
+
+```bash
+cp .env.example .env
+```
+
+2. Update the environment variables in the `.env` file with your own values. At a minimum, you'll need to provide:
+   - `DATABASE_URL`: Connection string for your PostgreSQL database
+   - `NEXTAUTH_SECRET`: A secure random string for session encryption
+   - NEXTAUTH_URL`: Your application's base URL (e.g., `http://localhost:3000`)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)